cli/flox/src/utils/init/logger.rs

```diff
@@ -83,8 +84,10 @@ pub fn init_logger(verbosity: Option<Verbosity>) {
         // Logs are being passed through by the `log` crate and correctly filtered by `tracing`.
         let filter = tracing_subscriber::filter::EnvFilter::try_new("trace").unwrap();
 
+        let use_colors = Dialog::can_prompt();
         let fmt_filtered = tracing_subscriber::fmt::layer()
             .with_writer(LockingTerminalStderr)
+            .with_ansi(use_colors)
```

Comment:
This is so that when you pipe logs to a file you don't get the ANSI escapes

Reply:
huh well spotted, `NO_COLOR` already works, i assumed the same thing would deal with terms as well..